### PR TITLE
Move configs to /configs, suffix TWIG rules with TwigRule to separate from phpstan rules by name

### PR DESCRIPTION
**Pull request by TomasVotruba**
Closes https://github.com/matthiasnoback/phpstan-twig-analysis/issues/24

### CODE REVIEW

**Comment by matthiasnoback** [comment]
Hey that was fast!

**Comment by matthiasnoback** [comment]
By the way, you can run `./qa` locally to fix everything before pushing

**Comment by matthiasnoback** [comment]
Only now do I understand why all the "tv"s in your work hahaha nice

**Comment by TomasVotruba** [comment]
> Only now do I understand why all the "tv"s in your work hahaha nice

:laughing:  I picked this convention from another project... the fastest git blame ever ;)

**Comment by TomasVotruba** [comment]
> By the way, you can run ./qa locally to fix everything before pushing

Neat... I just tried it :1st_place_medal:

**Comment by TomasVotruba** [comment]
Ready for 2nd shot :heavy_check_mark:

File: ecs.php [hunk 1]
```diff
@@ -15,8 +14,6 @@
     $ecsConfig->sets([SetList::CONTROL_STRUCTURES, SetList::PSR_12, SetList::COMMON, SetList::SYMPLIFY]);
 
     $ecsConfig->skip([
-        PhpUnitStrictFixer::class,
-        // Because it makes no sense ;) (well, I just need assertEquals())
```
[the remark this answers is not in the document]
Comment (matthiasnoback) [reply ecs.php:19]: Aha! Thanks :) You see, I just copied some stuff from another project...

**Comment by matthiasnoback** [comment]
Very happy with your contribution, thanks a lot!

**Comment by TomasVotruba** [comment]
Glad you like it 😊

**Comment by TomasVotruba** [comment]
Btw, I'm trying something similar today with Latte.
There is AST coming in latte/latte 3.0 any week now.

The API is almost 1:1 to Twig, so it will be easy to re-use rules 👍

**Comment by matthiasnoback** [comment]
Cool!

**Pull request merged**